assets/xml/samples/0 Sound Effects.xml

```diff
@@ -0,0 +1,1052 @@
+<Root>
+	<Sample Name="Bird Chirp 1">
+		<Offset Version="MQDebug" At="0"/>
+	</Sample>
+	<Sample Name="Insect Chirrup" SampleRate="16000">
+		<Offset Version="MQDebug" At="0x5b0"/>
```

Comment:
```suggestion
		<Offset Version="MQDebug" At="0x5B0"/>
```
We use capital notation for hex, there's a lot of xml's to update, sry :( regex will be your friend

Reply:
oof